cmd/console.go

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2023 The nebula-contrib Authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"log"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/nebula-contrib/ngctl/pkg/console"
+	"github.com/nebula-contrib/ngctl/pkg/util"
+)
+
+const consoleLabel = "ngctl/nebula-console"
+
+func consoleCmd() *cobra.Command {
+	var (
+		name      string
+		namespace string
+		image     string
+	)
+	cmd := &cobra.Command{
+		Use:   "console",
+		Short: "nebula console client for nebula graph ",
+		Long:  "nebula console client for nebula graph.",
+		Example: `  # init nebula graph console
+  operator-cli console init
+  # run nebula graph console
+  operator-cli console run
+`,
+	}
+
+	cmd.PersistentFlags().StringVar(&name, "name", "console", "name of the nebula graph console")
+	cmd.PersistentFlags().StringVar(&namespace, "namespace", "default", "namespace of the nebula graph")
+
+	init := cobra.Command{
+		Use:   "init",
+		Short: "init nebula graph console",
+		Long:  "init nebula graph console.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return initConsole(namespace, name, image)
+		},
+	}
+	init.PersistentFlags().StringVar(&image, "image", "vesoft/nebula-console:v3.5", "image of the nebula graph console")
+	cmd.AddCommand(&init)
+
+	run := cobra.Command{
+		Use:   "run",
+		Short: "run nebula graph console",
+		Long:  "run nebula graph console.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runConsole(namespace, name)
+		},
+	}
```

Comment:
can we combine two command, like ngctl console -u root -p nebula? and get addr and port by ./ngctl/config